core/test/audits/resource-summary-test.js

```diff
@@ -64,20 +64,6 @@ describe('Performance: Resource summary audit', () => {
       expect(thirdParty.transferSize).toBe(145);
       expect(thirdParty.requestCount).toBe(3);
     });
-
-    it('uses firstPartyHostnames if provided', async () => {
```

Comment:
FWIW this does seem like a feature we are losing with this PR. That being said, it's only used in this one hidden audit and not anywhere else so I still think it's fine.